srcpkgs/python-watchman/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'python-watchman'
+pkgname=python-watchman
+version=1.4.1
+revision=1
+wrksrc="pywatchman-${version}"
+build_style=python-module
+only_for_archs="x86_64 i686 x86_64-musl"
```

Comment:
no need for these

Reply:
just add python3-devel and python-devel to makedepends=

Reply:
this is great to know!